docker-compose.yml

```diff
@@ -6,53 +6,79 @@ services:
         image: mysql:5.6
         restart: always
         container_name: "akeneo_pim_mysql"
-        env_file: "./.env"
+        environment:
+            - "MYSQL_ROOT_PASSWORD=$MYSQL_ROOT_PASSWORD"
         command: "--max_allowed_packet=1000M --connect_timeout=72"
-        labels:
-            - com.pim.alias.mysql
         volumes:
-            - "./.mysql_data/mysql:/var/lib/mysql"
+            - "mysql_data:/var/lib/mysql"
             - "./conf/mysql/:/etc/mysql/conf.d"
+        ports:
+            - "3306:3306"
         networks:
             akeneo_network:
-                ipv4_address: 192.168.231.20
                 aliases:
                     - mysql.db
 
 
     # Akeneo Pim
     akeneo:
         build: .
-        image: pim/docker-akeneo-apline
+        image: pim/php-fpm-apline
         container_name: "akeneo_pim_app"
+        environment:
+            - "COMPOSER_CACHE_DIR=/cache"
         volumes:
             - "${SOURCE_PATH}:/var/www/pim/"
-            - "./scripts:/var/www/html/scripts"
-            - "./conf:/var/www/html/conf"
-        labels:
-            - com.pim.alias.web_app
+            - "./.composer/cache:/cache"
+        working_dir: /var/www/pim/
+        user: "$USER_ID"
         networks:
             akeneo_network:
-                ipv4_address: 192.168.231.10
                 aliases:
-                       - akeneo.pim
+                    - akeneo.pim
+
+    nginx:
+        image: nginx:alpine
+        container_name: "akeneo_pim_nginx"
+        volumes:
+            - "./conf/nginx/nginx-site.conf:/etc/nginx/conf.d/default.conf:ro"
```

Comment:
That will be next implementation, now splitting to `prod`, `dev`, and `behat`. Before it only had one host. :wink: